packages/utils/test/schema/getDefaultFormStateTest.ts

```diff
@@ -848,6 +862,52 @@ export default function getDefaultFormStateTest(testValidator: TestValidatorType
           requiredProperty: 'foo',
         });
       });
+      it('test an object const value populate as field defaults', () => {
+        const schema: RJSFSchema = {
+          type: 'object',
+          properties: {
+            localConst: {
+              type: 'string',
+              const: 'local',
+            },
+            RootConst: {
+              type: 'object',
+              properties: {
+                attr1: {
+                  type: 'number',
+                },
+                attr2: {
+                  type: 'boolean',
+                },
+              },
+              const: {
+                attr1: 1,
+                attr2: true,
+              },
+            },
+            RootAndLocalConst: {
+              type: 'string',
+              const: 'FromLocal',
+            },
+          },
+          const: {
+            RootAndLocalConst: 'FromRoot',
+          },
+        };
+        expect(
+          getObjectDefaults(testValidator, schema, {
+            rootSchema: schema,
+            experimental_defaultFormStateBehavior: { emptyObjectFields: 'skipDefaults' },
```

Comment:
Can you add at least 1 default to this schema just to make sure that it isn't included?

Reply:
I added one default but it will not be merged with the const/default value since we're testing the **getObjectDefaults** method. I added an additional test for the **getDefaultFormState** method where we merge the **formData** and the default value.